cassiopeia/tools/parameter_estimators.py

```diff
@@ -0,0 +1,426 @@
+"""
+This file stores functions for estimating lineage tracing parameters.
+Currently, we'll support the estimation of mutation and missing data rates.
+"""
+from cassiopeia.data.CassiopeiaTree import CassiopeiaTree
```

Comment:
This import (`cassiopeia.data.CassiopeiaTree`) needs to go down with the other Cassiopeia import.